db/migrate/20250105150012_create_course_material_text_chunk_references.rb

```diff
@@ -0,0 +1,24 @@
+class CreateCourseMaterialTextChunkReferences < ActiveRecord::Migration[7.2]
```

Comment:
Clean up migrations in d337e3329548c6739ceba762824554e34af7e34e, put them all together with e17e840d2185a9ae59d26d0e3a1345de1352be77.

Reply:
Squashed